DBMS-Session5-Assignment/src/queries/Queries.java

```diff
@@ -0,0 +1,48 @@
+package queries;
+
+//Class to store queries to be execute in the database 
+public class Queries {
+
+    // Select query to select items from database
+    /**
+     * Q-1 Given the id of a user, fetch all orders (Id, Order Date, Order
+     * Total) of that user which are in shipped state. Orders should be sorted
+     * by order date column in chronological order.
+     */
+    public static String querySelect = "SELECT op.order_id,o.order_date,SUM(p.cost*op.quantity) AS total_cost "
+            + "FROM orderitem o "
+            + "INNER JOIN orderproduct op "
+            + "ON op.order_id = o.id "
+            + "INNER JOIN product p "
+            + "ON p.id = op.product_id "
+            + "WHERE op.status IN('SHIPPED') "
+            + "AND o.user_id = ? GROUP BY op.order_id DESC";
+
+    // Insert query to insert images
+    /**
+     * Q-2 Insert five or more images of a product using batch insert technique.
+     */
+    public static String queryInsert = "INSERT INTO image(product_id,name,image)VALUES(?,?,?)";
+
+    // delete query to update products
+    /**
+     * Q-3 Delete all those products which were not ordered by any Shopper in
+     * last 1 year. Return the number of products deleted.
+     */
+    public static String update = "UPDATE product SET status = 'Inactive' "
+            + "WHERE id NOT IN ( SELECT DISTINCT product_id "
+            + "FROM orderproduct "
+            + "WHERE order_date > DATE_SUB(CURDATE(), INTERVAL 1 YEAR) )";
+

```

Comment:
Question is to "Delete all those products which were not ordered by any Shopper in last 1 year. Return the number of products deleted."

But you aren't getting those products which has never been ordered. This was missing in last assignment also. Review this and fix the query.

Hint -> You need to take last 1 year data from OrderProduct and join with products then get those products where OrderProduct's id is NULL.